Softeq.XToolkit.Common/Extensions/DateTimeExtensions.cs

```diff
@@ -29,6 +29,12 @@ public static DateTime LastDayOfWeek(this DateTime date)
             return lastDay;
         }
 
+        public static DateTime WithKind(this DateTime date, DateTimeKind kind)
```

Comment:
Could you please check `DateTime.SpecifyKind` method, looks like the same?

https://docs.microsoft.com/en-us/dotnet/api/system.datetime.specifykind?view=xamarinios-10.8